source/config.mts

```diff
@@ -0,0 +1,72 @@
+import path from "path";
```

Comment:
Any particular reason for the move from CoffeeScript? ~~I preferred that version, and it's more consistent with the rest of the code (e.g. `main.coffee`).~~ Ah, I see, but we want to move toward full typechecking, so this makes sense.